### DISCUSSION TITLE
How to allow pop up if cell has newline character?

### DISCUSSION THREAD

**Discussion by nhatton96** [Sequel-Ace/Sequel-Ace#1134] [Q&A]
If you turn off editing all fields with pop up sheet in Sequel Pro, pop up sheet is only used when there is a newline character.
But with Sequel Ace, it pops up whenever the field is text.
Is there a way that I can revert to the Sequel Pro interaction?

**Comment by Jason-Morcos** [comment]
See #1096

**Reply by nhatton96** [reply]
yeah this could be a work around if I make the switch length longer. But still switch at newline would be better.
